web/src/app/projects/repository/repository.component.html

```diff
@@ -17,6 +17,9 @@
   <mat-expansion-panel hideToggle #expansion>
     <mat-expansion-panel-header [collapsedHeight]="headerHeight+'px'" [expandedHeight]="headerHeight+'px'" fxLayout="row" fxLayout.lt-lg="column" fxLayoutAlign="space-between start">
       <div fxLayout="column" class="repository__container">
+        <div fxLayout="row">
+            <mat-progress-bar matTooltipPosition="above" matTooltip="Rating: {{rating | number:'1.0-2'}}%" mode="determinate" value={{rating}}></mat-progress-bar>
```

Comment:
Missing spacing `{{rating | number:'1.0-2'}}` to be `{{ rating | number:'1.0-2' }}`

Reply:
Value needs spacing but better with quotes and brackets `[value]="rating"`

Reply:
I will resolve 🤓